open-api/lang-tools/src/main/java/io/deephaven/lang/completion/ChunkerCompleter.java

```diff
@@ -784,7 +784,7 @@ private void invokeComplete(
         // for now, this will be a naive replacement, but later we'll want to look at _where_
         // in the invoke the cursor is; when on the ending paren, we'd likely want to look at
         // whether we are the argument to something, and if so, do a type check, and suggest useful .coercions().
-        String name = node.getName().trim();
+        String name = node.getName();
```

Comment:
Assuming this is just a cleanup?

Reply:
Yep - I was going to go further and remove name from a few spots, but just left it as not including `String name` as a param to the custom autocomplete interface - it isn't really needed, the invoke node will already trim on the way in/out.